datafusion/sqllogictest/test_files/copy.slt

```diff
@@ -469,8 +479,8 @@ select * from validate_arrow;
 # Error cases:
 
 # Copy from table with options
-query error DataFusion error: Invalid or Unsupported Configuration: Found unsupported option row_group_size with value 55 for JSON format!
-COPY source_table  to 'test_files/scratch/copy/table.json' (row_group_size 55);
+query error DataFusion error: Internal error: Config value "row_group_size" not found on JsonOptions.
```

Comment:
We automatically reject everything outside of the config struct. Since there is a single source of truth, maintaining the code becomes easier.